cluster/addons/rbac/cluster-autoscaler/cluster-autoscaler-rbac.yaml

```diff
@@ -0,0 +1,67 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cluster-autoscaler
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+rules:
+  # leader election
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["create"]
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    resourceNames: ["cluster-autoscaler"]
+    verbs: ["get", "update", "patch", "delete"]
+  # accessing & modifying cluster state (nodes & pods)
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["pods/eviction"]
+    verbs: ["create"]
+  # read-only access to cluster state, mostly around services & controllers
+  - apiGroups: [""]
+    resources: ["services", "replicationcontrollers"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["apps", "extensions"]
```

Comment:
Do we really need "extensions"? Didn't we already migrated everything to "apps" api group?

Reply:
No, the extensions group is very much alive :(

Reply:
Well - I know that the group itself is alive and available to users.
I'm just asking if cluster-autoscaler/scheduler is still using it (I thought we migrated usages of scheduler and cluster-autoscaler to already use apps api group - the underlying resource in the end is exactly the same).

Reply:
I can see code like ```Extensions().V1beta1().ReplicaSets()``` or ```ExtensionsV1beta1().DaemonSets``` in ClusterAutoscaler so I guess we're still using extensions.